SUIApp/SUIApp/View/ClimateView.swift

```diff
@@ -0,0 +1,386 @@
+                )
+            )
+            .rotationEffect(.degrees(-110))
+    }
+
+    private var climateView: some View {
+        ZStack {
+            Circle()
+                .fill(
+                    LinearGradient(
+                        colors: [.white, .white.opacity(0.2)],
+                        startPoint: .topLeading,
+                        endPoint: .bottomTrailing
+                    )
+                )
+                .frame(height: 250)
+                .opacity(0.06)
+                .blur(radius: 10)
+            Text(climateViewModel.updateClimate)
+                .font(.largeTitle)
+                .bold()
+                .foregroundColor(.white)
+                .frame(width: 150, height: 150)
+                .overlay(
+                    Circle()
+                        .stroke(
+                            gradient,
+                            lineWidth: 25
+                        )
+                )
+                .overlay(
+                    temperatureColorView
+                )
+        }
+    }
+
+    private var disclosureGroupView: some View {
+        DisclosureGroup {
+            sliderView(
+                title: Constants.acTitle,
+                imageName: Constants.snowImageName,
+                state: $climateViewModel.acSlider
+            )
+            sliderView(
+                title: Constants.fanTitle,
+                imageName: Constants.fanIconImageName,
+                state: $climateViewModel.fanSlider
+            )
+            sliderView(
+                title: Constants.heatTitle,
+                imageName: Constants.heatIconImageName,
+                state: $climateViewModel.heatSlider
+            )
+            sliderView(
+                title: Constants.autoTitle,
+                imageName: Constants.autoIconImageName,
+                state: $climateViewModel.autoSlider
+            )
+        } label: {
+            Text(Constants.settingTitle)
+        }
+        .padding()
+        .foregroundColor(.white)
+    }
+
+    private var dragGesture: some Gesture {
+        DragGesture()
+            .updating($gestureOffset) { value, state, _ in
+                state = value.translation
+                climateViewModel.onChangeMenuOffset(offset: gestureOffset)
+            }
+            .onEnded { _ in
+                let maxHeight = UIScreen.main.bounds.height - 100
+                withAnimation {
+                    climateViewModel.updateOffset(max: maxHeight)
+                }
+            }
+    }
+
+    private var bottomSheetView: some View {
+        VStack {
+            Capsule()
+                .fill(.black)
+                .frame(width: 60, height: 3)
+                .padding(.top)
+            visibaleBottomStackView
+                .padding(.horizontal)
+            Divider()
+            hiddenBottomSheetStackView
+            Spacer()
+        }
+        .frame(height: UIScreen.main.bounds.height - 100)
```

Comment:
```suggestion
        .frame(height: UIScreen.main.bounds.height - 100)
```
100 кто это?) Маг число?